hmf-common/src/main/java/com/hartwig/hmftools/common/linx/LinxDataLoader.java

```diff
@@ -303,13 +302,25 @@ private static Map<Integer, Integer> loadClusterExonCounts(@NotNull String filen
 class VisSvData
 {
     @NotNull
-    final Map<Integer, Integer> svIdToClusterId;
+    private final Map<Integer, Integer> svIdToClusterId;
     @NotNull
-    final Map<Integer, Integer> clusterIdToChainCount;
+    private final Map<Integer, Integer> clusterIdToChainCount;
 
     VisSvData(@NotNull Map<Integer, Integer> svIdToClusterId, @NotNull Map<Integer, Integer> clusterIdToChainCount)
     {
         this.svIdToClusterId = svIdToClusterId;
         this.clusterIdToChainCount = clusterIdToChainCount;
     }
+
+    @NotNull
+    public Map<Integer, Integer> getSvIdToClusterId()
+    {
+        return svIdToClusterId;
+    }
+
+    @NotNull
+    public Map<Integer, Integer> getClusterIdToChainCount()
+    {
+        return clusterIdToChainCount;
+    }
```

Comment:
We typically don't use "get" for getters since that is implicit. These functions should be named `svIdToClusterId()` and `clusterIdToChainCount()`